shell/platform/windows/keyboard_key_embedder_handler_unittests.cc

```diff
@@ -942,7 +942,10 @@ TEST(KeyboardKeyEmbedderHandlerTest,
       VK_NUMLOCK, kScanCodeNumLock, WM_KEYDOWN, 0, true, false,
       [&last_handled](bool handled) { last_handled = handled; });
   EXPECT_EQ(last_handled, false);
-  EXPECT_EQ(results.size(), 3);
+  // 4 total events should be fired:
+  // Pre-synchronization toggle, pre-sync press,
+  // main event, and post-sync press.
```

Comment:
You might as well expand it into detailed comparison of events. (Not necessarily for all fields, but to let people know what the events are.)